docs/docs/0.7.2/policies/external-services.md

```diff
@@ -0,0 +1,54 @@
+# External Service
+
+This policy enables consuming services that are not part of the mesh from services running inside it. The default passthrough cluster will allow for accessing any non-mesh host by its domain name or IP address, however as the name of the feature suggests, this pattern leaves the mesh administrator with no tools to aply any policies for such traffic. Therefore, ExternalService resource allows for declaring the desired external resource as a named service within the mesh and enabling the observability, security and traffic manipulation similar to any other service in the mesh.
```

Comment:
Would rewrite this paragraph:
- "...name or IP address, however as the name of the feature suggests, this pattern leaves the mesh administrator with no tools to aply any policies for such traffic." Would split the sentence before the "however" and leave out the "as the name suggests" since it is a bit wordy
- Confusing since it states that i have no tools to _apply_ any policies for such traffic but then you mention that you can enable observability, security, and traffic manipulation (also unclear what manipulation you are talking about)